modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js

```diff
@@ -78,17 +56,39 @@ const getDelimiter = (dateFormat) => {
 	return dateDelimiter;
 };
 
+const getLocaleDateFormat = (format = 'L') => {
+	return moment.localeData().longDateFormat(format);
+};
+
+const getMaskByDateFormat = (format) => {
+	const mask = [];
+
+	for (let i = 0; i < format.length; i++) {
+		if (/[a-z]/i.test(format[i])) {
+			mask.push(/\d/);
+		}
+		else {
+			mask.push(`${format[i]}`);
+		}
+	}
+
+	return mask;
+};
+
 const getDateFormat = () => {
-	const dateFormat = Liferay.AUI.getDateFormat();
-	const dateDelimiter = getDelimiter(dateFormat);
+	const dateMask = getLocaleDateFormat();
```

Comment:
@alinedoleron, why did you rename this variable? Shouldn't it still be named `dateFormat`?